.github/actions/fast-checkout/action.yml

```diff
@@ -32,7 +32,7 @@ runs:
       C_REPO: ${{ inputs.repository }}
       C_REF: ${{ inputs.ref }}
       C_PATH: ${{ inputs.path }}
-      C_TOKEN: ${{ inputs.token }}
+      GH_TOKEN: ${{ inputs.token }}
```

Comment:
I would put this either at the top or the bottom -- and it's not really a typographical error so "Typo" isn't the right commit message

Reply:
updated